### PR TITLE
Map

### PR DESCRIPTION
**Pull request by Franck59840**

### CODE REVIEW

File: assets/styles/columnRight.scss [hunk 1]
```diff
@@ -0,0 +1,166 @@
+/*CSS Colum review +Button add GooDeals*/
+@media only screen and (min-width: 1100px) {
+    h2{
```
Comment (Guillaum6e) [inline assets/styles/columnRight.scss:3]: Use a class please

File: assets/styles/blur.scss [hunk 2]
```diff
@@ -0,0 +1,3 @@
+body {
+    backdrop-filter: blur(5px);
+}
```
Comment (Guillaum6e) [inline assets/styles/blur.scss:3]: Is it necessary to have a new file just for this?
If it is used in the whole website, can you put it in app.scss?

File: assets/styles/columnRight.scss [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+    }
+   
+    ::-webkit-scrollbar {
+        display: none;
+    }
+}
+
+    @media only screen and (max-width: 1100px) {
+    
+        h2{
```
Comment (Guillaum6e) [inline assets/styles/columnRight.scss:94]: same here

File: assets/styles/columnRight.scss [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+            font-size: 1rem;
+        }
+       ul,li{
+            text-decoration: none;
```
Comment (Guillaum6e) [inline assets/styles/columnRight.scss:98]: same here

Comment (Franck59840) [reply assets/styles/columnRight.scss:98]: what utility create class for ul, li , just desactivited the text-decoration ?

File: assets/styles/blur.scss (hunk 2 above)
Comment (Franck59840) [reply assets/styles/blur.scss:3]: because the blur is not on all pages

**Review by Guillaum6e — CHANGES_REQUESTED** [review]
a few minor changes

File: assets/styles/columnRight.scss (hunk 4 above)
Comment (Franck59840) [reply assets/styles/columnRight.scss:98]: i'ts fix

**Review by Guillaum6e — APPROVED** [review]
nice

File: templates/about/about.html.twig [hunk 5]
```diff
@@ -0,0 +1,35 @@
+{% extends 'base.html.twig' %}
+
+{% block title %}
+    {{parent()}} - A propos
```
Comment (Benja-san) [inline templates/about/about.html.twig:4]: Optionnal : If "GoodealMap" is displayed in every pages, just put it above the head block inside base.html. by this process you won't have to write the {{ parent() }} expression in every pages.

**Review by Benja-san — APPROVED** [review]

**Pull request merged**